sail_on_client/protocol/localinterface.py

```diff
@@ -291,16 +291,19 @@ def evaluate(
             detection_idx = 1
             gt_detection_idx = metric.detection_id
             gt_classification_idx = metric.classification_id
+            novel_idx = 1
         # ######## Activity Recognition Evaluation  ###########
         elif domain == "activity_recognition":
             detection_idx = 1
             gt_detection_idx = metric.novel_id
             gt_classification_idx = metric.classification_id
+            novel_idx = 89
         # ######## Document Transcription Evaluation  ###########
         elif domain == "transcripts":
             detection_idx = 1
             gt_detection_idx = metric.novel_id
             gt_classification_idx = metric.classification_id
+            novel_idx = 50
         else:
             raise AttributeError(
                 f'Domain: "{domain}" is not a real domain.  Get a clue.'
```

Comment:
Could you add an incline comment to the three things you are setting here (like why do these need to be set for each experiment?  

Also, looks like you just need to set novel_idx, the rest of these are identical.   Maybe switch out for just want is necessary here?

Reply:
Addressed in c976557 and 979b64a